src/layer/transportation_label.js

```diff
@@ -127,7 +127,11 @@ export const bridgeSpacer = {
   type: "symbol",
```

Comment:
There’s also a `text-field` value above that relies on string interpolation instead of `concat` and `get` expressions. Converting to the latter will facilitate automated localization.